### DISCUSSION TITLE
Why do we need to install it globally?

### DISCUSSION THREAD

**Discussion by itsmebasti** [coreybutler/node-windows#370] [General]
I had some issues with the dependencies, so I have a quick question:

When node-windows is installed globally, my IDE does not resolve the dependency and there is no IDE support (everything works fine though). Linking didn't help (I'm using yarn2). So I decided to install it in the package (only). This worked locally, but on the Windows server I got an error, that was already reported [here](https://github.com/coreybutler/node-windows/issues/57) saying `Error: Cannot find module 'node-windows'`

Why is the global installation recommended and why is the server unable to resolve the dependency?

**Comment by coreybutler** [comment]
Does [this](https://github.com/coreybutler/node-windows#:~:text=The%20myappname.exe,recommended%20installation%20instructions.) answer your question?
